### DISCUSSION TITLE
Add shortcut to force-open mod settings when selecting mods that don't require it

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#18185] [Ideas]
~~At the moment, mods from the "conversion" and "fun" sections are not bound to the keys on the keyboard.~~

~~At the same time, mods from the "reducing" difficulty occupy only 4 keys of the first row of the keyboard. It would be great to have 6 more first row keys under this category, since these mods are used quite often in everyday life (unlike fun mods).~~

Additionally, can give the ability to open the mod's customization window immediately if you select it while holding down CTRL + mouse click (or the button on the keyboard, V for DT, for example).

**Comment by frenzibyte** [comment]
Improvement of keyboard navigation is tracked in https://github.com/ppy/osu/issues/5623. Reworded thread to match the other proposal.

**Reply by Moltanicaa** [reply]
I think the title is a little confusing. It is written that "that don't require it." Perhaps it would be more correct "for which customization is possible"?

**Reply by frenzibyte** [reply]
We have the concept of mods which require customisation/configuration (e.g. difficulty adjust) and mods which don't (any other mod which supports customisation). 

Mod settings open when selecting a mod that requires customisation, but doesn't when selecting a mod that doesn't require it.

**Comment by peppy** [comment]
I propose right-clicking should do this.

**Reply by Moltanicaa** [reply]
I also suggest and "keyboard only" method (Ctrl + mod button)

**Reply by peppy** [reply]
More like ctrl+click should right click regardless.

**Reply by Moltanicaa** [reply]
If we continue this topic, then for touch screens can make a long tap (~0.5 seconds) in mod to quickly open customization.

**Reply by frenzibyte** [reply]
Yes, that's known. See https://github.com/ppy/osu/issues/6567.
